composer.json

```diff
@@ -23,7 +23,7 @@
     "require": {
         "php": ">=5.5.0",
         "ext-mongodb": "*",
-        "mongodb/mongodb": "~1.1.1"
+        "mongodb/mongodb": "~1.1|~1.2"
```

Comment:
@rossaffandy This constraint does not appear to be valid. You could accomplish a similar requirement via:

```
"mongodb/mongodb": ">=1.1.1 <1.3"
```